app/lib/llm_clients/openai/azure_client.rb

```diff
@@ -0,0 +1,32 @@
+require "openai"
+
+module LlmClients
+  module Openai
+    class AzureClient < Client
+      private
+
+      def client_provider
+        "openai_azure"
+      end
+
+      def chat_client
+        az_client(@model)
+      end
+
+      def embed_client
+        az_client(@embedding_model)
+      end
+
+      def az_client(model_name)
+        @az_client ||= OpenAI::Client.new(
+          access_token: @api_key, #ENV.fetch("AZURE_OPENAI_API_KEY"),
```

Comment:
Remove comment?

I agree we shouldn't use an env var, since a single API key doesn't really fit with the ModelServer concept.